### ISSUE TITLE
Create budget template for Paris 2019

### ISSUE THREAD

**Issue by bmann** [ethereum-magicians/scrolls#43]
I cloned the Prague one, still need to do edits https://docs.google.com/spreadsheets/d/1PKimcwUvPqtjn02-jL07gq3KyDLKD3-i1K5qd6pRXg0/edit#gid=3

Talk to Jerome about:

- [x] video and/or livestream
- [x] catering
- [x] venue costs

**Comment by ethernian** [comment]
### Venue costs
@jdetychey have proposed 2-3 amphitheaters, but we have preferred a one big flat room for many working groups with few smaller private rooms for private conversations. This is because we will not few speakers for big audience, but many (ad-hoc) working circles. It is possible too.

### Video and/or livestream costs
How it depends on venue? Consider we have a big room with flexible room partitioning and with some adhoc working groups inside.  How many cameras an camera teams will we have there?
 
Is there some kind of formula, like `RecordedHours * PricePerRecordedHour  + NumOfCameraMen *  PricePerWorkHour`  ?

**Comment by bmann** [comment]
Still need venue costs regardless of layout. 

Video: I would recommend just one crew to cover the “main stage”. I can estimate until we get some quotes, likely $2K USD for the day. If ETHCC already has a crew, catering, etc may need to extend for a day. 

This is assigned to me, will follow up with Jerome.

**Comment by bmann** [comment]
OK, I have done a first pass at the budget https://docs.google.com/spreadsheets/d/1PKimcwUvPqtjn02-jL07gq3KyDLKD3-i1K5qd6pRXg0/edit#gid=3

Note: I have no idea what costs actually are, so those are just estimates.

Knowing how many people will attend is a big driver of costs -- as catering is the big one.

We might just do lunch to save a bit of money, but I've kept coffee + lunch in the budget for now.

I have also modeled having people pay $25 to attend. I think this will also help with attendance.
